### DISCUSSION TITLE
Accessibility: add alternative text to different objects when knitting to formats like Word, PDF

### DISCUSSION THREAD

**Discussion by IZE85** [quarto-dev/quarto-cli#5511] [Q&A]
In MS Word, one is able to "add alternative text to a shape, picture, chart, SmartArt graphic, or other object to help people with visual disabilities."

When I add alternative text to a figure via "fig-alt" it gets exported via HTML. Unfortunately, when knitting to formats like Word or PDF, my fig-alt text is not saved/deposited somewhere, if I'm correct. At least, for MS Word, I cannot see the alternative text in the "alternative text" field, MS Word provides for figures.

My question: Is it possible for the fig-alt text to find its way into the MS Word Alternative text field and have something similar for PDFs?

Thank you in advance!

Ad alternative text:
"Alt text helps people with visual disabilities understand pictures and other graphical content. When someone using a screen reader comes across a picture in a document, they will hear the alt text describing the picture; without alt text, they will only know they've reached a picture without knowing what the picture shows."
Source: https://support.microsoft.com/en-us/office/add-alternative-text-to-a-shape-picture-chart-smartart-graphic-or-other-object-44989b2a-903c-4d9a-b742-6a75b451c669

**Comment by mcanouil** [comment]
I opened an issue (see #5514), because alt-text should work in Quarto as it does with Pandoc.

Regarding PDF, I don't know how to check alt-text in a PDF (at least in MacOS Preview), so was unable to verify if Pandoc template already handle alt-text or not.
Note that the issue for PDF could be upstream, _i.e._, in Pandoc (see https://github.com/jgm/pandoc/issues/5409).

**Comment by cderv** [comment]
I think we have an issue in the sense that we should indeed handles our `fig-alt` attributes on image. 

However, you can use Pandoc syntax (https://pandoc.org/MANUAL.html#images) which work for docx output 

````markdown
---
title: doc
format: docx
---

## Section

![Caption for Quarto Logo](https://quarto.org/quarto.png "Some alt text")
````

Not great though because this would be a Title in HTML format

For PDF, I don't know what the concept of Alternative text if working. If you have some LaTeX example on how to handle it, we'll be interested.

**Reply by mcanouil** [reply]
See #5514 😉

**Reply by cderv** [reply]
yes see my answer there that I posted before 😉
